manual/src/cmds/intf-c.etex

```diff
@@ -942,7 +942,7 @@ void foo (value v1, value v2, value v3)
 {
   CAMLparam3 (v1, v2, v3);
   ...
-  CAMLreturn0;
+  CAMLreturn(Val_unit);
 }
 \end{verbatim}
```

Comment:
This should not be changed. It illustrates the rule that function having parameters or local variables of type `value` should use the macros `CAMLreturn*` and `CAMLparam*`. Such functions are allowed to have return type `void`, so long as they are not called from OCaml. Note that the change would be a C type error, since we are returning a `value` from a `void` function.